### PR TITLE
RCPS-115:Prefect System: Create OffenseFacadeImplTest unit test

### PR DESCRIPTION
**Pull request by katieann7**
Description

The class OffenseFacadeImplTest should have proper unit tests.

also include:
RCPS-114:Prefect System: Create CommunityServiceFacadeImplTest unit test
Description

The class CommunityServiceFacadeImplTest should have proper unit tests.

### CODE REVIEW

**Comment by RoCS2024** [comment]
Build is failing. moving back to In Progress

**Comment by katieann7** [comment]
Done po sir.

File: build.xml [hunk 1]
```diff
@@ -79,7 +79,7 @@
 
     <!-- Generate java documentation   -->
     <target name="generate-javadoc">
-        <javadoc packagenames="com.prefect.office.record.management.*"
+        <javadoc packagenames="com.prefect.office.recorda.management.*"
```
Comment (RoCS2024) [inline build.xml:82]: typo

File: src/main/java/Main.java [hunk 2]
```diff
@@ -19,8 +17,8 @@
 
 public class Main {
     private static final Scanner scanner = new Scanner(System.in);
-    private static final OffenseFacade offenseFacade = new OffenseFacadeImpl(new OffenseDaoImpl());
-    private static final CommunityServiceFacade communityServiceFacade = new CommunityServiceFacadeImpl(new CommunityServiceDaoImpl());
+    private static final OffenseFacade offenseFacade = new OffenseFacadeImpl();
```
Comment (RoCS2024) [inline src/main/java/Main.java:20]: why are these objects final?

File: src/main/java/com/prefect/office/record/management/appl/facade/prefect/communityservice/impl/CommunityServiceFacadeImpl.java [hunk 3]
```diff
@@ -7,9 +7,9 @@
 import java.util.List;
 
 public class CommunityServiceFacadeImpl implements CommunityServiceFacade {
-    private final CommunityServiceDao communityServiceDao;
+    private CommunityServiceDao communityServiceDao;
 
-    public CommunityServiceFacadeImpl(CommunityServiceDao communityServiceDao) {
+    public CommunityServiceFacadeImpl() {
         this.communityServiceDao = communityServiceDao;
```
Comment (RoCS2024) [inline src/main/java/com/prefect/office/record/management/appl/facade/prefect/communityservice/impl/CommunityServiceFacadeImpl.java:13]: this dao is null

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

File: src/main/java/Main.java (hunk 2 above)
Comment (katieann7) [reply src/main/java/Main.java:20]: About po this one. I double check po yung last merge naka final na po sya. Tinanggal ko lng po yung parameter for unit testing.

**Comment by katieann7** [comment]
Done po sir.

**Review by RoCS2024 — APPROVED** [review]

**Pull request merged**